framework/Source/Operations/AddBlend.metal

```diff
@@ -2,15 +2,23 @@
 #include "OperationShaderTypes.h"
 using namespace metal;
 
+typedef struct
+{
+    // 0 - 1
+    float opacity;
+} AddBlendUniform;
+
 fragment half4 addBlendFragment(TwoInputVertexIO fragmentInput [[stage_in]],
-                                     texture2d<half> inputTexture [[texture(0)]],
-                                     texture2d<half> inputTexture2 [[texture(1)]])
+                                texture2d<half> inputTexture [[texture(0)]],
+                                texture2d<half> inputTexture2 [[texture(1)]],
+                                constant AddBlendUniform& uniform [[ buffer(1) ]])
 {
     constexpr sampler quadSampler;
     half4 base = inputTexture.sample(quadSampler, fragmentInput.textureCoordinate);
     constexpr sampler quadSampler2;
     half4 overlay = inputTexture2.sample(quadSampler, fragmentInput.textureCoordinate2);
-    
+    overlay.rgb *= uniform.opacity;
```

Comment:
question: 

Applying this to the RGB color, I'm not sure how it would affect the algorithm below. Is it okay to just update the return color?

```swift
return half4(r, g, b, a * uniform.opacity);
```

Reply:
quick consult with @Justin-Ruan 

If we apply value here, it's more control blend strength. He suggests to name it `intensity`

Reply:
Hi @yyjim, I have renamed the opacity to the intensity and will merge this PR if you don't have other concerns.
Also, for the version tag:
 - By Semantic Versioning，it might be 1.1.0 since the PR is made to add a new feature (intensity adjustment) instead of bug fixing
 - But I worry the change is too small to increase a minor version

Reply:
I'm fine with updating the minor version